MediaBrowser.Model/FileOrganization/EpisodeFileOrganizationRequest.cs

```diff
@@ -13,5 +15,38 @@ public class EpisodeFileOrganizationRequest
         public int? EndingEpisodeNumber { get; set; }
 
         public bool RememberCorrection { get; set; }
+        public string NewSeriesName { get; set; }
+
+        public string NewSeriesYear { get; set; }
+
+        public string NewSeriesProviderIds { get; set; }
+
+        public string TargetFolder { get; set; }
+
+        public Dictionary<string, string> NewSeriesProviderIdsDictionary
```

Comment:
I guess I misunderstood you. 
You're just referring to having both NewSeriesProviderIds and NewSeriesProviderIdsDictionary, right?